mobile/screens/Analytics/AnalyticsDashboardScreen.tsx

```diff
@@ -44,7 +90,55 @@ const styles = StyleSheet.create({
     flex: 1,
     justifyContent: "flex-start",
     backgroundColor: "#f2f2f2",
+    flexDirection: "row",
+  },
+  analyticsContainer: {
+    flex: 1,
+    marginRight: 3,
+    padding: 10,
+    borderRadius: 12,
+    backgroundColor: "#fff",
+    height: 130,
     flexDirection: "column",
-    margin: 3,
+  },
+  graphContainer: {
+    flex: 1,
+    marginLeft: 3,
+    padding: 10,
+    borderRadius: 12,
+    alignItems: "center",
+    backgroundColor: "#fff",
+    height: 130,
+  },
+  boxTitle: {
+    fontFamily: "DMSans-Bold",
+    color: "grey",
+    fontSize: 10,
+  },
+  sideBySideText: {
+    paddingTop: 10,
+    flexDirection: "row",
+  },
+  bigBlueText: {
+    fontFamily: "DMSans-Bold",
+    color: "blue",
+    fontSize: 22,
+    justifyContent: "flex-start",
+  },
+  mediumGrayText: {
+    fontFamily: "DMSans-Bold",
+    color: "grey",
+    fontSize: 16,
+    justifyContent: "flex-start",
+    paddingTop: 5,
+    bottom: 0,
+    right: 0,
+  },
+  bottomRightTextFont: {
```

Comment:
These styles need to be responsive. Hardcoding values do not look correct on all screen sizes.